clear
python run.py 2> log_error.txt
